modules/dxp/apps/commerce/commerce-machine-learning-impl/build.gradle

```diff
@@ -25,11 +25,11 @@ dependencies {
 	compileOnly project(":apps:dispatch:dispatch-api")
 	compileOnly project(":apps:frontend-taglib:frontend-taglib")
 	compileOnly project(":apps:info:info-api")
-	compileOnly project(":apps:petra:petra-json-web-service-client")
 	compileOnly project(":apps:portal-search:portal-search-api")
 	compileOnly project(":apps:portal-search:portal-search-engine-adapter-api")
 	compileOnly project(":apps:portal-vulcan:portal-vulcan-api")
 	compileOnly project(":apps:portal:portal-instance-lifecycle-api")
+	compileOnly project(":apps:portal:portal-json-web-service-client")
```

Comment:
@julschong No direct reference to the packages exported by portal-json-web-service-client, we should remove this compileOnly.

Tina.

Reply:
pushed again with it removed